### ISSUE TITLE
Python: isConnected() returns True after the scanner is turned off / in stand-by

### ISSUE THREAD

**Issue by Contused** [clariusdev/cast#157]
I recently noticed that isConnected() does not work how I thought it would behave.

Behaviour:
-I connect to the scanner with cast.connect(ip, 5828, "research")
-I check for cast.isConnected() status every 10sec. in a thread
-If the ultrasound probe is going to stand-by or is turned off by holding down the power button: the app shows "Casting information not available". To my understanding the connection is closed.

Expected:
If I check with isConnected() I should get True if there is an active connection between scanner and my software
If I check with isConnected() I should get False if there is no active connection between scanner and my software

What happens instead:
After the scanner is turned off/ in stand-by and the app shows "Casting information not available" I still receive True when checking cast.isConnected().
Also sometimes during scanning I receive False when checking cast.isConnected() in a seperate thread.

Data:
Device: C3HD3 Firmware 11.2.0
App: 11.2.0
Python Software CAST: 11.2.0 (cast.dll, cast.lib), pyclariuscast.pyd from last available release

Question:
Is this a bug or am I wrong on the expected behaviour?

**Comment by Contused** [comment]
Maybe the Issue can be resolved with a 11.2. pyclariuscast.pyd? The release files are missing the pyd.

**Comment by julien-l** [comment]
Hi we do not provide `.pyd` files, we provide `.so` files in the release artifacts. I will try to reproduce the issue.

**Comment by p3p31v** [comment]
Any progress with this? I'm having the same issue

**Comment by clariusk** [comment]
no, we have not investigated this issue, but will look to see if it's still reproducable
